src/content/en/updates/2017/07/devtools-release-notes.md

```diff
@@ -0,0 +1,394 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New features and changes coming to DevTools in Chrome 61.
+
+{# wf_updated_on: 2017-07-10 #}
+{# wf_published_on: 2017-07-10 #}
+{# wf_tags: chrome61,devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: New features and changes coming to DevTools in Chrome 61. #}
```

Comment:
I like to explicitly mention that the doc only includes new features and major changes... Some people expect release notes to also include bug fixes